backend/package.json

```diff
@@ -13,8 +13,10 @@
     "dev": "node build-assets.js --watch",
     "check:eslint": "npx eslint --ignore-path ../.gitignore static/js",
     "fix:eslint": "npx eslint --fix --ignore-path ../.gitignore static/js",
-    "check:stylelint": "npx stylelint --ignore-pattern uswds static/scss",
-    "fix:stylelint": "npx stylelint --fix --ignore-pattern uswds static/scss",
+    "check:stylelint": "npx stylelint '**/*.{css,scss,sass}'",
+    "fix:stylelint": "npx stylelint --fix '**/*.{css,scss,sass}'",
+    "check:prettier": "npx prettier --check .",
```

Comment:
Prettier is currently checking a lot of stuff that we haven't checked in the past (as far as I can tell, anyway). Rendered JSON files, yaml, markdown, etc. 

I'm not sure if we want to start checking those things or handle them in other ways.